source/index.html

```diff
@@ -1,12 +1,207 @@
+        <li class="about-list__item">Увеличивает лайки на фотографиях</li>
+        <li class="about-list__item">Связано вручную с любовью и умилением</li>
+        <li class="about-list__item">Поддержка отечественного производителя</li>
+        <li class="about-list__item">Поставляется в подарочной упаковке</li>
+      </ul>
+    </section>
+
+    <section class="reviews">
+      <h2 class="reviews__title">Отзывы</h2>
+      <div class="review">
+        <blockquote class="review__content">
+          Я молодая мама, но всё равно фотографии не набирали и близко такого количества лайков, как у популярных
+          инстамамочек. В отчаянии, я накупила аксессуаров и игрушек в Мишке, и мои фотографии сразу стали более
+          стильными, а также
+          набирают больше лайков!
+        </blockquote>
+        <cite class="review__author">Анастасия Красильникова</cite>
+        <p class="review__author-link">@misssssiskras</p>
+      </div>
+      <div class="reviews__button-group">
+        <button class="reviews__button-prev" type="button">Предыдущий</button>
+        <button class="reviews__button-next" type="button">Следующий</button>
+      </div>
+      <a class="reviews__button button" href="#">Написать</a>
+    </section>
+
+    <section class="contacts">
+      <h2 class="contacts__title">Контакты</h2>
+      <dl class="contacts__list">
+        <dt class="contacts__item">e-mail:</dt>
+        <dd class="contacts__email">
+          <a href="mailto:info@mimimishkashop.ru">info@mimimishkashop.ru</a>
+        </dd>
+        <dt class="contacts__item">адрес:</dt>
+        <dd class="contacts__address">г. Санкт-Петербург ул. Большая Конюшенная д. 19/8, офис 101</dd>
+      </dl>
+
+      <div class="contacts__map">
+        <img class="contacts__map-image" src="img/content/map-mobile.jpg" alt="">
+      </div>
+      <a class="contacts__button button" href="#">Напишите нам</a>
+    </section>
+  </main>
+
+  <footer class="page-footer">
+    <ul class="social social-list">
+      <li class="social-list__item">
+        <a class="social-list__link social-list__link--instagram" href="https://www.instagram.com/htmlacademy">Мы в
+          Инстаграме</a>
+      </li>
+      <li class="social-list__item">
+        <a class="social-list__link social-list__link--facebook" href="https://www.facebook.com/htmlacademy">Мы в
+          Фейсбуке</a>
+      </li>
+      <li class="social-list__item">
+        <a class="social-list__link social-list__link--twitter" href="https://twitter.com/htmlacademy_ru">Мы в
+          Твиттере</a>
+      </li>
+    </ul>
+
+    <div class="page-footer__copyright copyright">
+      <p class="copyright__text">Разработано:</p>
+      <a class="copyright__link" href="https://htmlacademy.ru/intensive/adaptive">
+        <span class="visually-hidden">Логотип HTML Academy</span>
+        <svg class="copyright__logo" width="27" height="35" viewBox="0 0 27 35" fill="none">
```

Comment:
```suggestion
        <svg class="copyright__logo" width="27" height="35" viewBox="0 0 27 35" fill="none" aria-labelledby="title">
```
это надо же сказать, что свг описывается этим тайтлом с этим id